lib/hawkular/inventory/inventory_api.rb

```diff
@@ -99,33 +112,34 @@ def list_resources_for_feed(feed_id, fetch_properties = false, filter = {})
     # @param [Boolean] fetch_properties Shall additional runtime properties be fetched?
     # @return [Array<Resource>] List of resources. Can be empty
     def list_resources_for_type(resource_type_path, fetch_properties = false)
-      path = resource_type_path.is_a?(CanonicalPath) ? resource_type_path : CanonicalPath.parse(resource_type_path)
-      resource_type_id = path.resource_type_id
-      feed_id = path.feed_id
-      if feed_id.nil?
-        ret = http_get("/traversal/rt;#{resource_type_id}/rl;defines/type=r")
-      else
-        ret = http_get("/traversal/f;#{feed_id}/rt;#{resource_type_id}/rl;defines/type=r")
-      end
-      ret.map do |r|
-        if fetch_properties && !feed_id.nil?
-          p = get_config_data_for_resource(r['path'])
-          r['properties'] = p['value']
-        end
-        Resource.new(r)
-      end
+      path = CanonicalPath.parse_if_string(resource_type_path)
+      fail 'Feed id must be given' unless path.feed_id
+      fail 'Resource type must be given' unless path.resource_type_id
+
+      # Fetch metrics by tag
+      feed_path = feed_cp(path.feed_id)
```

Comment:
I think you need to unescape the `feed_id` or make `feed_cp` use it as-it. It should be already escaped since it comes from a CP. This is making MiQ to fail when WF servers are in domain mode.